backends/__init__.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+__all__ = ['AbstractBackend', 'FileBackend']
+
+class AbstractBackend(object):
+
+    def __init__(self, *args, **kwargs):
+        super(AbstractBackend, self).__init__(*args, **kwargs)
```

Comment:
I think you can get rid of this entire `__init__` method. AFAIK, Python doesn't require subclasses of `object` to call `__init__` on the superclass.